src/Release/Entity/PublicRelease.php

```diff
@@ -0,0 +1,12 @@
+<?php
+
+namespace Clearbooks\Labs\Release\Entity;
+/**
+ * Created by PhpStorm.
+ * User: Volodymyr
+ * Date: 17/08/2015
+ * Time: 16:37
+ */
+interface PublicRelease extends \Clearbooks\Labs\Release\Common\PublicRelease
```

Comment:
Right I'm confused here, this empty Public release interface extends the \Common\PublicRelease? Why do we need both?